src/components/Profile/My posts/MyPostForm.jsx

```diff
@@ -25,7 +25,7 @@ const MyPostForm = ({ handleSubmit, reset, addPost }) => {
           name="newPost"
           placeholder="Enter your post"
           label="NewPost"
-          validate={[required, maxLenght10]}
+          validate={[required, maxLength10]}
```

Comment:
I think this is an extra - maybe will look nice if you use maxLengthCreator inside?
`validate={[required, maxLengthCreator(10)]}`